packages/vite/src/node/shortcuts.ts

```diff
@@ -112,6 +112,13 @@ const BASE_SHORTCUTS: CLIShortcut[] = [
       openBrowser(url, true, server.config.logger)
     },
   },
+  {
+    key: 'c',
+    description: 'clear console',
+    async action(server) {
+      server.config.logger.clearScreen('error')
```

Comment:
Shouldn't this be `silent` so that it always clear the console?
```suggestion
      server.config.logger.clearScreen('silent')
```

Reply:
I think when I want to clear the screen, it is completely clear

Reply:
If you think silent is more appropriate, there is no problem